operators/pkg/controller/elasticsearch/configmap/configmap_control.go

```diff
@@ -0,0 +1,46 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package configmap
+
+import (
+	"reflect"
+
+	"github.com/elastic/cloud-on-k8s/operators/pkg/apis/elasticsearch/v1alpha1"
+	"github.com/elastic/cloud-on-k8s/operators/pkg/controller/common/reconciler"
+	"github.com/elastic/cloud-on-k8s/operators/pkg/utils/k8s"
+
```

Comment:
We try to keep only two groups in imports: std libs and third parties. But it is not easy if you are using IntelliJ ;)

Reply:
Fixed.